src/model/element.js

```diff
@@ -198,6 +198,16 @@ export default class Element extends Node {
 		return nodes;
 	}
 
+	getNodeByPath( relativePath ) {
```

Comment:
I missed that I added this method completely. Good catch :) Tests are needed too.